index.html

```diff
@@ -0,0 +1,147 @@
+                <li class="sub-nav-item"><i class="fa-brands fa-twitter"></i></li>
+                <li class="sub-nav-item">English</li>
+                <li class="sub-nav-item">My Page</li>
+                <li class="sub-nav-item">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="navbar fixed-top-nav" id="top-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-icon"></i>
+            <div class="desktop-nav">
+                <img class="logo-nav" src="./images/aspire-logo.png" width="110px" alt="Aspire logo">
+                <ul class="nav-list float-right">
+                    <li><a class="menu-list-item" href="about.html">About</a></li><br>
+                    <li><a class="menu-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <nav class="mobile-nav">
+        <span class="close-button">x</span>
+        <ul class="menu-list-mobile">
+            <li><a class="menu-list-item" href="about.html">About</a></li><br>
+            <li><a class="menu-list-item" href="#main-program">Program</a></li><br>
+            <li><a class="menu-list-item" href="#join">Join</a></li><br>
+            <li><a class="menu-list-item" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="menu-list-item" href="#news">News</a></li><br>
+            <li><a href="#" class="mt-3  p-2" id="btn-nav">BP Launch</a></li><br>
+        </ul>
+    </nav>
+
+    <main class="main-section">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally.</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <h3 class="message-time">24/7 Active Services</h3>
+                    <p>@Rwanda Kigali converntion center</p>
+                </div>
```

Comment:
- [ ] Dear @Arsenethierry, good job making adjustments to your tour section, But I notice some of the required changes mentioned by the previous reviewer are not yet addressed. the first thing is we need to use the same font as the original design and the original design uses `Fonts: Cocogoose, Lato` for the font type. Please use this font type for a better layout. 
- [ ] Also as suggested earlier please adjust the spacing to match the original design. You can see there is a large spacing between the top and the bottom of the main section. Please remember as frontend engineers we need to be consistent with the original design. 

### Original design 
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/69077061/3b088282-b630-4d51-a024-ec87afcd23ce)

### Your design 
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/69077061/993bc6e3-d67c-4f31-8e33-f2cda0e67248)